src/Std/Data/DHashMap/Basic.lean

```diff
@@ -285,6 +285,11 @@ but will later become a primitive operation.
   ⟨(Raw₀.insertMany ⟨m.1, m.2.size_buckets_pos⟩ l).1,
    (Raw₀.insertMany ⟨m.1, m.2.size_buckets_pos⟩ l).2 _ Raw.WF.insert₀ m.2⟩
 
+@[inline, inherit_doc Raw.insertList] def insertList (m : DHashMap α β)
```

Comment:
I changed that. It remains open how to name the lemmas. I kept ```insertList``` for now in the name. Using just ```insertMany``` seems impractical in case someone verifies insertList for arrays, trees etc. ```insertMany_list``` seemed a bit long

Reply:
I think this is resolved :)